.nvmrc

```
@@ -0,0 +1 @@
+20.14.0
```

NAB, we updated to 20.15.1 recently everywhere
```suggestion
20.15.1
```